tests/stp/stripe/stripe.csv

```diff
@@ -0,0 +1,45 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+before,0,0,q,.servers.startup[],1,,"Get connection management set up"
+before,0,0,q,system "sleep 1",1,,"Wait a sec"
+before,0,0,q,rdbHandles:rdbNames!gethandle each rdbNames:exec procname from (.proc`readprocs).proc`file where proctype=`rdb,1,,"Open handle to RDBs"
+true,0,0,q,numSeg~sum `rdb=.servers.SERVERS`proctype,1,,"Check the number of running rdbs is the same as that of process.csv"
+before,0,0,q,stpHandle:gethandle[`stp1],1,,"Open handle to STP"
+before,0,0,q,stpHandle(`.stplg.init;testlogdb),1,,"Create test stplog test directory"
+before,0,0,q,currlog:stpHandle"currlog",1,,"Get STP log table"
+before,0,0,q,met:stpHandle".stpm.metatable",1,,"Get STP meta table"
+before,0,0,q,value[rdbHandles] @\: "trade:0#trade;quote:0#quote",1,,"Clear rdbs tables"
+before,0,0,q,"t1:(value[rdbHandles] @\: ""count trade"")",1,,"Get initial trade table counts"
+before,0,0,q,"q1:(value[rdbHandles] @\: ""count quote"")",1,,"Get initial quote table counts"
+before,0,0,q,![-11;] each exec logname from currlog where tbl in `trade`quote,1,,"Replay logs locally"
+before,0,0,q,t2:count trade,1,,"Get local trade table count"
+before,0,0,q,q2:count quote,1,,"Get local quote table count"
+before,0,0,q,{delete from x} each `trade`quote,1,,"Clear local tables"
+run,0,0,q,row:100000,1,,"Rows to publish"
+run,0,0,q,"stpHandle @/: `.u.upd ,/: ((`trade;t row);(`quote;q row);(`heartbeat;`error))",1,,"Publish table updates"
+run,0,0,q,system "sleep 1",1,,"Wait for update to publish"
+run,0,0,q,![-11;] each exec logname from currlog where tbl in `trade`quote`segmentederrorlogfile,1,,"Replay logs locally"
+true,0,0,q,"all not raze value exec null logname,null handle from currlog",1,,"Check lognames and handles generated"
+true,0,0,q,currlog[`trade;`logname]~exec last logname from met where `trade in/: tbls,1,,"Check lognames the same in log and meta"
+true,0,0,q,currlog[`quote;`logname]~exec last logname from met where `quote in/: tbls,1,,"Check lognames the same in log and meta"
+true,0,0,q,(0#trade)~(exec first schema from met where `trade in/: tbls)`trade,1,,"Check trade schema saved to meta table"
+true,0,0,q,(0#quote)~(exec first schema from met where `quote in/: tbls)`quote,1,,"Check quote schema saved to meta table"
+true,0,0,q,"(row+sum[t1],sum q1)~sum(value rdbHandles) @\: ""count[trade],count quote""",1,,"Check trade and quote updates were correctly published"
```

Comment:
If I'm reading this test right, it will check that a total of 100000 rows have been added to the tables across all the RDBs - this doesn't necessarily mean that the updates have been published correctly though, right?

e.g. I think we have 4 rdbs & syms should be roughly evenly distributed between them so we'd expect ~25000 rows added per rdb, but if all 100000 went to rdb1 and 0 to the other 3, I think this test would still pass?

Might be better to look at the distinct syms present in each rdb & check there's no overlap or something like that?

Reply:
The additional check for updates have been published correctly is in line 38, i.e. there is no overlap of syms.

- `correct` checks that the sym updates went to the correct rdb 
- `notwrong` checks that there is not any sym update to the specific rdb that belongs to the other (wrong) rdbs

Line 39 iterates through a list of different rows of update, and checks everything is correct based on the accumation of the appropriate previous tests.
i.e. sum is correct, and syms are correct.

https://github.com/AquaQAnalytics/TorQ/blob/0d1e6a828fb1dccc625ca135324efdcfa2f18932/tests/stp/stripe/stripe.csv#L38-L39

Will work on the additional test for the distribution count

Reply:
Added the additional test for the distribution count in 08fa7c6e8289ff69a67d84bcf2b3a7c0dc0236b9